tests/common/logout_test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var App = require('../lib/app');
+var assert = require('assert');
+
+describe('Logout', function() {
+  var app;
+  var loginView;
+  var servicesView;
+  this.timeout(30000);
+
+  before(() => {
+    app = new App();
+    return app.init()
+      .then(defaultView => { loginView = defaultView; });
+  });
+
+  after(() => {
+    return app.stop();
+  });
+
+  it('should logout', () => {
+    return loginView.loginSuccess(12345678).then(function(servicesView) {
```

Comment:
Sorry thought I could not use the first .then inside the first arrow function. The first example is the same like it was but there is one more then. I understand know what you mean, in case there were more how it is best to handle them.
Thanks!